> Can I trouble you to try and pull in [9a4d9eb](https://github.com/NixOS/nixpkgs/commit/9a4d9ebd74a7b5754e470ca1eb97f9677e081b7b) to see if that sorts out zeal-qt6?

Sadly still fails with
```
zeal> CMake Error at src/app/CMakeLists.txt:36 (target_link_libraries):
zeal>   Error evaluating generator expression:
zeal>     $<TARGET_PROPERTY:Qt6::QmlIntegration,_qt_module_has_private_headers>
zeal>   Target "Qt6::QmlIntegration" not found.
```
But the relevant cmake files are indeed in qtdeclarative.

--- a/pkgs/development/libraries/qt-6/fetch.sh
+++ b/pkgs/development/libraries/qt-6/fetch.sh
@@ -1,1 +1,1 @@
-WGET_ARGS=( https://download.qt.io/official_releases/qt/6.4/6.4.3/submodules/ -A '*.tar.xz' )
+WGET_ARGS=( https://download.qt.io/official_releases/qt/6.5/6.5.0/submodules/ -A '*.tar.xz' )

--- a/pkgs/development/libraries/qt-6/hooks/qtbase-setup-hook.sh
+++ b/pkgs/development/libraries/qt-6/hooks/qtbase-setup-hook.sh
@@ -54,9 +54,12 @@ else # Only set up Qt once.
         dontPatchMkspecs=1
 
         local lib="${!outputLib}"
+        local dev="${!outputDev}"
 
-        if [ -d "$lib/mkspecs/modules" ]; then
-            fixQtModulePaths "$lib/mkspecs/modules"
+        moveToOutput "mkspecs/modules" "$dev"
+
+        if [ -d "$dev/mkspecs/modules" ]; then
+            fixQtModulePaths "$dev/mkspecs/modules"
         fi
 
         if [ -d "$lib/mkspecs" ]; then

--- a/pkgs/development/libraries/qt-6/modules/qtbase.nix
+++ b/pkgs/development/libraries/qt-6/modules/qtbase.nix
@@ -244,7 +244,8 @@ stdenv.mkDerivation rec {
   moveToDev = false;
 
   postFixup = ''
-    fixQtModulePaths  "$out/mkspecs/modules"
+    moveToOutput      "mkspecs/modules" "$dev"
+    fixQtModulePaths  "$dev/mkspecs/modules"
     fixQtBuiltinPaths "$out" '*.pr?'
   '';
 

--- a/pkgs/development/libraries/qt-6/patches/0001-qtbase-qmake-always-use-libname-instead-of-absolute-.patch
+++ b/pkgs/development/libraries/qt-6/patches/0001-qtbase-qmake-always-use-libname-instead-of-absolute-.patch
@@ -1,4 +1,4 @@
-From 8880bc263a366aeb82056f0bf3f1b17b6ec26900 Mon Sep 17 00:00:00 2001
+From 69d9faa9e4420d3cb0d1466c1b95ceadb2cd75f3 Mon Sep 17 00:00:00 2001
 From: Nick Cao <nickcao@nichi.co>
 Date: Thu, 13 Apr 2023 23:42:29 +0800
 Subject: [PATCH 1/6] qtbase: qmake: always use libname instead of absolute
@@ -14,10 +14,10 @@ paths can be incorrect.
  2 files changed, 4 insertions(+), 6 deletions(-)
 
 diff --git a/cmake/QtFinishPrlFile.cmake b/cmake/QtFinishPrlFile.cmake
-index 32169e418c..4e754af367 100644
+index 1cf9377e6ce..ac4428bd7a1 100644
 --- a/cmake/QtFinishPrlFile.cmake
 +++ b/cmake/QtFinishPrlFile.cmake
-@@ -61,9 +61,10 @@ foreach(line ${lines})
+@@ -64,9 +64,10 @@ foreach(line ${lines})
                          endif()
                          list(APPEND adjusted_libs "-framework" "${CMAKE_MATCH_1}")
                      else()
@@ -32,10 +32,10 @@ index 32169e418c..4e754af367 100644
                  endif()
              else()
 diff --git a/cmake/QtGenerateLibHelpers.cmake b/cmake/QtGenerateLibHelpers.cmake
-index e3f4bbf881..f8bd26acc7 100644
+index 3ffe354fd8d..441332d4582 100644
 --- a/cmake/QtGenerateLibHelpers.cmake
 +++ b/cmake/QtGenerateLibHelpers.cmake
-@@ -70,9 +70,6 @@ function(qt_transform_absolute_library_paths_to_link_flags out_var library_path_
+@@ -73,9 +73,6 @@ function(qt_transform_absolute_library_paths_to_link_flags out_var library_path_
              string(TOLOWER "${dir}" dir_lower)
              # If library_path isn't in default link directories, we should add it to link flags.
              list(FIND IMPLICIT_LINK_DIRECTORIES_LOWER "${dir_lower}" index)

--- a/pkgs/development/libraries/qt-6/patches/0002-qtbase-qmake-fix-mkspecs-for-darwin.patch
+++ b/pkgs/development/libraries/qt-6/patches/0002-qtbase-qmake-fix-mkspecs-for-darwin.patch
@@ -1,6 +1,6 @@
-From 034db4e75ec749ac78fcf8235fa659b0eca83c30 Mon Sep 17 00:00:00 2001
+From 41e32c41f781261726722628122c924abb532575 Mon Sep 17 00:00:00 2001
 From: Nick Cao <nickcao@nichi.co>
-Date: Fri, 14 Apr 2023 09:34:08 +0800
+Date: Fri, 14 Apr 2023 21:43:04 +0800
 Subject: [PATCH 2/6] qtbase: qmake: fix mkspecs for darwin
 
 ---
@@ -13,7 +13,7 @@ Subject: [PATCH 2/6] qtbase: qmake: fix mkspecs for darwin
  6 files changed, 1 insertion(+), 415 deletions(-)
 
 diff --git a/mkspecs/common/mac.conf b/mkspecs/common/mac.conf
-index 61bea952b2..9909dae726 100644
+index 61bea952b22..9909dae7260 100644
 --- a/mkspecs/common/mac.conf
 +++ b/mkspecs/common/mac.conf
 @@ -23,7 +23,7 @@ QMAKE_INCDIR_OPENGL     = \
@@ -26,7 +26,7 @@ index 61bea952b2..9909dae726 100644
  
  QMAKE_LFLAGS_REL_RPATH  =
 diff --git a/mkspecs/features/mac/default_post.prf b/mkspecs/features/mac/default_post.prf
-index 09db1764b1..aadfce875e 100644
+index 4acf3b19d5c..aadfce875e2 100644
 --- a/mkspecs/features/mac/default_post.prf
 +++ b/mkspecs/features/mac/default_post.prf
 @@ -1,9 +1,5 @@
@@ -155,7 +155,7 @@ index 09db1764b1..aadfce875e 100644
 -    QMAKE_XCODE_ARCHS += $$QMAKE_APPLE_DEVICE_ARCHS
 -    QMAKE_MAC_XCODE_SETTINGS += arch_device
 -
--    simulator {
+-    ios:simulator {
 -        arch_simulator.name = "ARCHS[sdk=$${simulator.sdk}*]"
 -        arch_simulator.value = $$QMAKE_APPLE_SIMULATOR_ARCHS
 -        QMAKE_XCODE_ARCHS += $$QMAKE_APPLE_SIMULATOR_ARCHS
@@ -168,7 +168,7 @@ index 09db1764b1..aadfce875e 100644
 -    QMAKE_MAC_XCODE_SETTINGS += only_active_arch
 -} else {
 -    device|!simulator: VALID_DEVICE_ARCHS = $$QMAKE_APPLE_DEVICE_ARCHS
--    simulator: VALID_SIMULATOR_ARCHS = $$QMAKE_APPLE_SIMULATOR_ARCHS
+-    ios:simulator: VALID_SIMULATOR_ARCHS = $$QMAKE_APPLE_SIMULATOR_ARCHS
 -    VALID_ARCHS = $$VALID_DEVICE_ARCHS $$VALID_SIMULATOR_ARCHS
 -
 -    single_arch: VALID_ARCHS = $$first(VALID_ARCHS)
@@ -207,7 +207,7 @@ index 09db1764b1..aadfce875e 100644
 -    # and makes it easier for people to override EXPORT_VALID_ARCHS to limit
 -    # individual rules to a different set of architecture(s) from the overall
 -    # build (such as machtest in QtCore).
--    simulator:device {
+-    ios:simulator:device {
 -        QMAKE_XARCH_CFLAGS =
 -        QMAKE_XARCH_LFLAGS =
 -        QMAKE_EXTRA_VARIABLES += QMAKE_XARCH_CFLAGS QMAKE_XARCH_LFLAGS
@@ -245,7 +245,7 @@ index 09db1764b1..aadfce875e 100644
 -        QMAKE_CXXFLAGS += $(EXPORT_QMAKE_XARCH_CFLAGS)
 -        QMAKE_LFLAGS += $(EXPORT_QMAKE_XARCH_LFLAGS)
 -    } else {
--        simulator {
+-        ios:simulator {
 -            version_identifier = $$simulator.deployment_identifier
 -            platform_identifier = $$simulator.sdk
 -            sysroot_path  = $$xcodeSDKInfo(Path, $$simulator.sdk)
@@ -313,7 +313,7 @@ index 09db1764b1..aadfce875e 100644
      generate_xcode_project.commands = @$(QMAKE) -spec macx-xcode \"$(EXPORT__PRO_FILE_)\" $$QMAKE_ARGS
      generate_xcode_project.target = xcodeproj
 diff --git a/mkspecs/features/mac/default_pre.prf b/mkspecs/features/mac/default_pre.prf
-index e3534561a5..3b01424e67 100644
+index e3534561a56..3b01424e67b 100644
 --- a/mkspecs/features/mac/default_pre.prf
 +++ b/mkspecs/features/mac/default_pre.prf
 @@ -1,60 +1,2 @@
@@ -378,7 +378,7 @@ index e3534561a5..3b01424e67 100644
 -xcode_copy_phase_strip_setting.value = NO
 -QMAKE_MAC_XCODE_SETTINGS += xcode_copy_phase_strip_setting
 diff --git a/mkspecs/features/mac/sdk.mk b/mkspecs/features/mac/sdk.mk
-index a32ceacb6c..e69de29bb2 100644
+index a32ceacb6ce..e69de29bb2d 100644
 --- a/mkspecs/features/mac/sdk.mk
 +++ b/mkspecs/features/mac/sdk.mk
 @@ -1,27 +0,0 @@
@@ -410,7 +410,7 @@ index a32ceacb6c..e69de29bb2 100644
 -    endif
 -endif
 diff --git a/mkspecs/features/mac/sdk.prf b/mkspecs/features/mac/sdk.prf
-index 3a9c2778bb..e69de29bb2 100644
+index 3a9c2778bbe..e69de29bb2d 100644
 --- a/mkspecs/features/mac/sdk.prf
 +++ b/mkspecs/features/mac/sdk.prf
 @@ -1,61 +0,0 @@
@@ -476,7 +476,7 @@ index 3a9c2778bb..e69de29bb2 100644
 -    cache($$tool_variable, set stash, $$tool)
 -}
 diff --git a/mkspecs/features/mac/toolchain.prf b/mkspecs/features/mac/toolchain.prf
-index df191eb13c..e69de29bb2 100644
+index df191eb13c4..e69de29bb2d 100644
 --- a/mkspecs/features/mac/toolchain.prf
 +++ b/mkspecs/features/mac/toolchain.prf
 @@ -1,5 +0,0 @@

--- a/pkgs/development/libraries/qt-6/patches/0003-qtbase-qmake-fix-includedir-in-generated-pkg-config.patch
+++ b/pkgs/development/libraries/qt-6/patches/0003-qtbase-qmake-fix-includedir-in-generated-pkg-config.patch
@@ -1,4 +1,4 @@
-From bc91f05db85b774f26d6bce86e2e618dfc7a6883 Mon Sep 17 00:00:00 2001
+From f52f3c2cb1703592eaeb43e80f585a24ce8402d7 Mon Sep 17 00:00:00 2001
 From: Nick Cao <nickcao@nichi.co>
 Date: Fri, 14 Apr 2023 09:34:46 +0800
 Subject: [PATCH 3/6] qtbase: qmake: fix includedir in generated pkg-config
@@ -8,10 +8,10 @@ Subject: [PATCH 3/6] qtbase: qmake: fix includedir in generated pkg-config
  1 file changed, 1 insertion(+), 2 deletions(-)
 
 diff --git a/qmake/generators/makefile.cpp b/qmake/generators/makefile.cpp
-index da585bd9b2..3abf9cee83 100644
+index cc985a878b4..4e3b383d812 100644
 --- a/qmake/generators/makefile.cpp
 +++ b/qmake/generators/makefile.cpp
-@@ -3402,8 +3402,7 @@ MakefileGenerator::writePkgConfigFile()
+@@ -3403,8 +3403,7 @@ MakefileGenerator::writePkgConfigFile()
        << varGlue("QMAKE_PKGCONFIG_CFLAGS", "", " ", " ")
          //      << varGlue("DEFINES","-D"," -D"," ")
           ;

--- a/pkgs/development/libraries/qt-6/patches/0004-qtbase-fix-locating-tzdir-on-NixOS.patch
+++ b/pkgs/development/libraries/qt-6/patches/0004-qtbase-fix-locating-tzdir-on-NixOS.patch
@@ -1,4 +1,4 @@
-From d612c1d7161f95864b9383df84b16d8c24fbcc9b Mon Sep 17 00:00:00 2001
+From dd0dfc9cf87966f5d7493a943ec04c665be83cb6 Mon Sep 17 00:00:00 2001
 From: Nick Cao <nickcao@nichi.co>
 Date: Fri, 14 Apr 2023 09:35:25 +0800
 Subject: [PATCH 4/6] qtbase: fix locating tzdir on NixOS
@@ -8,7 +8,7 @@ Subject: [PATCH 4/6] qtbase: fix locating tzdir on NixOS
  1 file changed, 17 insertions(+), 10 deletions(-)
 
 diff --git a/src/corelib/time/qtimezoneprivate_tz.cpp b/src/corelib/time/qtimezoneprivate_tz.cpp
-index e87e34f76d..39bd79d4a4 100644
+index 960a0944185..a5186acbd91 100644
 --- a/src/corelib/time/qtimezoneprivate_tz.cpp
 +++ b/src/corelib/time/qtimezoneprivate_tz.cpp
 @@ -51,7 +51,11 @@ typedef QHash<QByteArray, QTzTimeZone> QTzTimeZoneHash;
@@ -24,7 +24,7 @@ index e87e34f76d..39bd79d4a4 100644
      if (!QFile::exists(path))
          path = QStringLiteral("/usr/lib/zoneinfo/zone.tab");
  
-@@ -729,18 +733,21 @@ QTzTimeZoneCacheEntry QTzTimeZoneCache::findEntry(const QByteArray &ianaId)
+@@ -730,18 +734,21 @@ QTzTimeZoneCacheEntry QTzTimeZoneCache::findEntry(const QByteArray &ianaId)
          if (!tzif.open(QIODevice::ReadOnly))
              return ret;
      } else {

--- a/pkgs/development/libraries/qt-6/patches/0005-qtbase-deal-with-a-font-face-at-index-0-as-Regular-f.patch
+++ b/pkgs/development/libraries/qt-6/patches/0005-qtbase-deal-with-a-font-face-at-index-0-as-Regular-f.patch
@@ -1,4 +1,4 @@
-From 5bd3672c7870b2e46e2a734dc9a9cb1837375a1c Mon Sep 17 00:00:00 2001
+From 4e8c14f1af9c332826e0454f4fd63e541edbaf5c Mon Sep 17 00:00:00 2001
 From: Nick Cao <nickcao@nichi.co>
 Date: Tue, 21 Mar 2023 15:48:49 +0800
 Subject: [PATCH 5/6] qtbase: deal with a font face at index 0 as Regular for
@@ -10,10 +10,10 @@ Reference: https://bugreports.qt.io/browse/QTBUG-111994
  1 file changed, 1 insertion(+)
 
 diff --git a/src/gui/text/unix/qfontconfigdatabase.cpp b/src/gui/text/unix/qfontconfigdatabase.cpp
-index 9b60cf2963..5a42ef6a68 100644
+index 474644b871f..c7a117fd134 100644
 --- a/src/gui/text/unix/qfontconfigdatabase.cpp
 +++ b/src/gui/text/unix/qfontconfigdatabase.cpp
-@@ -554,6 +554,7 @@ void QFontconfigDatabase::populateFontDatabase()
+@@ -556,6 +556,7 @@ void QFontconfigDatabase::populateFontDatabase()
              FcObjectSetAdd(os, *p);
              ++p;
          }

--- a/pkgs/development/libraries/qt-6/patches/0006-qtbase-qt-cmake-always-use-cmake-from-path.patch
+++ b/pkgs/development/libraries/qt-6/patches/0006-qtbase-qt-cmake-always-use-cmake-from-path.patch
@@ -1,4 +1,4 @@
-From f0017e872297168ab616096180891c7f312ef1a1 Mon Sep 17 00:00:00 2001
+From 61ae6e04388dd40e11c214d56f22f8f2007bf35f Mon Sep 17 00:00:00 2001
 From: Nick Cao <nickcao@nichi.co>
 Date: Wed, 12 Apr 2023 10:13:50 +0800
 Subject: [PATCH 6/6] qtbase: qt-cmake: always use cmake from path
@@ -10,7 +10,7 @@ during the build of qtbase, bloating the runtime closure of qtbase.
  1 file changed, 1 insertion(+), 6 deletions(-)
 
 diff --git a/bin/qt-cmake.in b/bin/qt-cmake.in
-index f719257f60..571ffe788f 100755
+index f719257f602..571ffe788fa 100755
 --- a/bin/qt-cmake.in
 +++ b/bin/qt-cmake.in
 @@ -4,12 +4,7 @@

--- a/pkgs/development/libraries/qt-6/srcs.nix
+++ b/pkgs/development/libraries/qt-6/srcs.nix
@@ -4,283 +4,307 @@
 
 {
   qt3d = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qt3d-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0w9xmsrd3mqbm5vf1m8cv67kcjrcbjnfmm6fmw2icg95jzwjb0m8";
-      name = "qt3d-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qt3d-everywhere-src-6.5.0.tar.xz";
+      sha256 = "05h84cdsicdg71sx4v9s8vx98i2xh2n7n02wxkxivwj468151ci0";
+      name = "qt3d-everywhere-src-6.5.0.tar.xz";
     };
   };
   qt5compat = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qt5compat-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0ymak3cr36b8hyr3axxywrv153ds4kcj8p04x7p7bm93p2mlkcnl";
-      name = "qt5compat-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qt5compat-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0q8jq9ccb0as7qhxqgs1q84i7qxz3xx6wbqsn0qy3hiz34xgbqm9";
+      name = "qt5compat-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtactiveqt = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtactiveqt-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0v8wf7xv5dqcw9v75a1zhhfqhmrya9q66az3awkfscjda78y2gh9";
-      name = "qtactiveqt-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtactiveqt-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0fwwz2ag4s03kicmgkpvgg3n6glx2ld21b24xqi3ib5av75smc15";
+      name = "qtactiveqt-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtbase = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtbase-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0q0si40bmgbplczr1skacd98zkfh6mmigax4q71pnphnn3jwk1sh";
-      name = "qtbase-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtbase-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1vzmxak112llvnx9rdgss99i9bc88rzsaxn59wdyqr5y9xxsmqgx";
+      name = "qtbase-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtcharts = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtcharts-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0in36za9iq41mc1hq62vjd8zni6amdd2b24gqngzcpdmzzsy8qaa";
-      name = "qtcharts-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtcharts-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1a165qz40yc50wdzk9sz0va6nc92y280x3k6yw8y0vgmlx81vkgw";
+      name = "qtcharts-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtconnectivity = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtconnectivity-everywhere-src-6.4.3.tar.xz";
-      sha256 = "17acli5wksd793v145mv8a4ld59v8g9dvv32wxlyvdsarha2137r";
-      name = "qtconnectivity-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtconnectivity-everywhere-src-6.5.0.tar.xz";
+      sha256 = "01lgd6bv23zgj0c788h4ii192mf8cvcm2d5jfwd3d1mrp99ncqz7";
+      name = "qtconnectivity-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtdatavis3d = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtdatavis3d-everywhere-src-6.4.3.tar.xz";
-      sha256 = "15brg1gcx2am3wbr54lx20fw1q42gryjxxnxf600nmk3nrfsqy69";
-      name = "qtdatavis3d-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtdatavis3d-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0sw1m61md30n06whl7s1d8ylr24pxjqs4q66a617vbp72xvw32nl";
+      name = "qtdatavis3d-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtdeclarative = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtdeclarative-everywhere-src-6.4.3.tar.xz";
-      sha256 = "15d73d957zfhls3ny322i1n9iqvg2nxk8swi00v5w4w8p6rx3pk7";
-      name = "qtdeclarative-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtdeclarative-everywhere-src-6.5.0.tar.xz";
+      sha256 = "15wcb2zq4sl1aw8yc1np9bp2p0df4r9in3zks3d9255wiv6k3mpp";
+      name = "qtdeclarative-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtdoc = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtdoc-everywhere-src-6.4.3.tar.xz";
-      sha256 = "10w12bsfwmxw6z4n50mv653q6vj7bcb7r0pmik52kxi9sr6w7skk";
-      name = "qtdoc-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtdoc-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1fblc7yr2gxmwi325lv3pwfkbbdrk2i4564y4fwdahl58xncwqi6";
+      name = "qtdoc-everywhere-src-6.5.0.tar.xz";
+    };
+  };
+  qtgrpc = {
+    version = "6.5.0";
+    src = fetchurl {
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtgrpc-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1wrgrr58lyg3g8dc5a3qbl7p0ym6k6g9zkly0kwz6pbbihv4p7sq";
+      name = "qtgrpc-everywhere-src-6.5.0.tar.xz";
     };
   };
   qthttpserver = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qthttpserver-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0yf162pxm55aybm62z1qqf3h9ff39iy72ffpxk775fbrqynxqyn3";
-      name = "qthttpserver-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qthttpserver-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0mnmaz333prww2b5vxjix4zlm1pgi2snavpqbg4swprvh93pc3b7";
+      name = "qthttpserver-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtimageformats = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtimageformats-everywhere-src-6.4.3.tar.xz";
-      sha256 = "165pk7z2k0ymzkm1r8fjykc6hlxdrpc2b0ysqlbldf3l5q35izqa";
-      name = "qtimageformats-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtimageformats-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0y3g0i11nfrg1h1d7jnnckky6hnfrx7z6cysq0r03rn77b6i1y7r";
+      name = "qtimageformats-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtlanguageserver = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtlanguageserver-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0xbrsg1z9p9wwx1zhh9birb44lb8ri1c6afjlv2cf69f8h31120f";
-      name = "qtlanguageserver-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtlanguageserver-everywhere-src-6.5.0.tar.xz";
+      sha256 = "078siwgsb1ypiim869jdkkmp32g715kkc76fj6764id3yg9d7j4d";
+      name = "qtlanguageserver-everywhere-src-6.5.0.tar.xz";
+    };
+  };
+  qtlocation = {
+    version = "6.5.0";
+    src = fetchurl {
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtlocation-everywhere-src-6.5.0.tar.xz";
+      sha256 = "036351d5yb35fma1wpvh6zcrbcsfg97ks6hy67vlbbsq958aggqf";
+      name = "qtlocation-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtlottie = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtlottie-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0gb9y4c9d1x548hpvcjbgr0pvw3v4c1vicqy6ppavv368ph54v7z";
-      name = "qtlottie-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtlottie-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0ip2nx169pvrxrpw1viivh20sq96c29z7njvk18nqsi8p7gfq9c4";
+      name = "qtlottie-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtmultimedia = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtmultimedia-everywhere-src-6.4.3.tar.xz";
-      sha256 = "003xav0vxlh6i6l0nk9m7ikaa86nfxk2xarjw2gfb89dw5lj99x4";
-      name = "qtmultimedia-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtmultimedia-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0im1visfb2r88pcrx7sb80znl17cij9l1qwr93n1ml5x7b3x104l";
+      name = "qtmultimedia-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtnetworkauth = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtnetworkauth-everywhere-src-6.4.3.tar.xz";
-      sha256 = "1rbaf73ijvr6y91scdyk5cjnsm930yj2ck2gnvxwif7lfajmn4ad";
-      name = "qtnetworkauth-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtnetworkauth-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0gj14j50d50fl0rjwilss4nakvxwldbg3iz5kdnbwvhkn8m55k6v";
+      name = "qtnetworkauth-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtpositioning = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtpositioning-everywhere-src-6.4.3.tar.xz";
-      sha256 = "036pph2hy2wzr1z6gs3zc688zxifnkc001p9ba9y44kwsghv265j";
-      name = "qtpositioning-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtpositioning-everywhere-src-6.5.0.tar.xz";
+      sha256 = "11n86llfixrgqw7yqxr1fcspq0khmyiwiwmibacbmlnrpwg159qd";
+      name = "qtpositioning-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtquick3d = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtquick3d-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0l40vkada3l1zkz042lcg9ybkqd3bg6wlc0vzngr76s4bmb8v8vq";
-      name = "qtquick3d-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtquick3d-everywhere-src-6.5.0.tar.xz";
+      sha256 = "19bj6xzw63rwbzlj7lb0hbni37syfyiyzwjdxj6crdcqr8lh8ncv";
+      name = "qtquick3d-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtquick3dphysics = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtquick3dphysics-everywhere-src-6.4.3.tar.xz";
-      sha256 = "05fv5mbcfqzmrr3ciqlx3vw5b6agk3kpb4r548h0hcacqjiyi1mb";
-      name = "qtquick3dphysics-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtquick3dphysics-everywhere-src-6.5.0.tar.xz";
+      sha256 = "04f0k2z3sqpbkjn74ral0q5s2hkkzijnr5ay9f91c6lg4ciaki81";
+      name = "qtquick3dphysics-everywhere-src-6.5.0.tar.xz";
+    };
+  };
+  qtquickeffectmaker = {
+    version = "6.5.0";
+    src = fetchurl {
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtquickeffectmaker-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1mwwn8psmq0lbv1ggc4g42ns93ygg6yqd555aq7qkxyhr9rcccjb";
+      name = "qtquickeffectmaker-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtquicktimeline = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtquicktimeline-everywhere-src-6.4.3.tar.xz";
-      sha256 = "199xbvjq1xg1lzkkq4ilbp1jiikiqg9khbzijz3ribx3qd3w821q";
-      name = "qtquicktimeline-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtquicktimeline-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1gygaxb9p6d23q5nxmhjlmazzx4i3vkhvjsi9v6l7d32js93x2sp";
+      name = "qtquicktimeline-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtremoteobjects = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtremoteobjects-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0shq1i26k76nymvlj48l5n4afn06j6kbca463lclk8nbg7glg54w";
-      name = "qtremoteobjects-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtremoteobjects-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1bn24l46ia0nvvcbzs5h3wg5nlk94m35fqrv1lcl8km8mzkvch7z";
+      name = "qtremoteobjects-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtscxml = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtscxml-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0fignzvz9wc37s94bdnqd7z8x6a5m3adbiz32gkh4k23dl0jqwpy";
-      name = "qtscxml-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtscxml-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0vkn2p4w21z9a6q27hf4yda85hjs4s01wdxy47b7cjngp0y888gi";
+      name = "qtscxml-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtsensors = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtsensors-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0j5wp93hlf1gpb9y55llad9pimjz20hp5w5xl0v6fic953x68faz";
-      name = "qtsensors-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtsensors-everywhere-src-6.5.0.tar.xz";
+      sha256 = "19ifwxbsa0k2p7z4wa1q4g4shi668w9wprhxkcp2sz4iyki39r2y";
+      name = "qtsensors-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtserialbus = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtserialbus-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0rj3nfs017vmp8i7f1hg2mrav7fkwh6cby803ib4xw6i2rsnli5n";
-      name = "qtserialbus-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtserialbus-everywhere-src-6.5.0.tar.xz";
+      sha256 = "15s82ic6w5jw8c1xnwwmskchynvcazmfq4ai7kfrz764ca9kfj4p";
+      name = "qtserialbus-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtserialport = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtserialport-everywhere-src-6.4.3.tar.xz";
-      sha256 = "1plmzkn1g00g0vrw5n9kawq1y6fj0cgbryrr5a59m8zgcy8av5sz";
-      name = "qtserialport-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtserialport-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1zsd9aas8p7zgfjx2fji0mfn6fzy6822g0h52lxdyjlajzssj2cj";
+      name = "qtserialport-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtshadertools = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtshadertools-everywhere-src-6.4.3.tar.xz";
-      sha256 = "1y384xw3jb1x4z3qzwzjxp7ymg20qn4sb4i7sq5s4sg7wd6bfj66";
-      name = "qtshadertools-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtshadertools-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0xapkzvz79wspc1a9l6yvp7m2vsfmrvapdsymkvz2w9hgw1qsqc6";
+      name = "qtshadertools-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtspeech = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtspeech-everywhere-src-6.4.3.tar.xz";
-      sha256 = "1qja4n2wkkxkcczr1afi8d083qq4lrngkvj698w1s1habqcx1q3r";
-      name = "qtspeech-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtspeech-everywhere-src-6.5.0.tar.xz";
+      sha256 = "16ixx1b8r5v5m04jpjylixpl4xw1qh5g0dp4phj40vg5z8vjg08x";
+      name = "qtspeech-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtsvg = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtsvg-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0jlshycc0cy3ja652g6jb51p4q31dsxfsz28brq9h67qdj45ycc8";
-      name = "qtsvg-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtsvg-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0kvs0sb32r4izskh17l771z9lfmmk6951q5lrf5y4ladyihpxjk4";
+      name = "qtsvg-everywhere-src-6.5.0.tar.xz";
     };
   };
   qttools = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qttools-everywhere-src-6.4.3.tar.xz";
-      sha256 = "14d0qmqdyrz524srb5iwn5j2fm136582bs32zs7axlswrllzhzc6";
-      name = "qttools-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qttools-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1i39cdl0mmf0wxmkmq16jx1mnj8s7p7bhsa2jnz8hjd4n2b3vhs9";
+      name = "qttools-everywhere-src-6.5.0.tar.xz";
     };
   };
   qttranslations = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qttranslations-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0b4pprdczbnk1gvda2bs1fg84yinii9ih201m2l4k5nl01w6prbr";
-      name = "qttranslations-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qttranslations-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0gbs0shf9hm0xrj3n3zkfdpdymks2wa11nna7ijiixckhgyx11gw";
+      name = "qttranslations-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtvirtualkeyboard = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtvirtualkeyboard-everywhere-src-6.4.3.tar.xz";
-      sha256 = "1r8fvqjmh18x89snxflzci1vinf7jvflfjihidffc02vdwi8aiiz";
-      name = "qtvirtualkeyboard-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtvirtualkeyboard-everywhere-src-6.5.0.tar.xz";
+      sha256 = "19qg59yhln7hbny6nfaz9wx4cm8ng3j23y3snpsfj5q84iwdwibv";
+      name = "qtvirtualkeyboard-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtwayland = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtwayland-everywhere-src-6.4.3.tar.xz";
-      sha256 = "12a7pi39zn7miyli6ywhkfx7vh0sl2h5iddp226f80acizd63cf6";
-      name = "qtwayland-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtwayland-everywhere-src-6.5.0.tar.xz";
+      sha256 = "18sr2yijlm4yh13kq8v7l1knp6b01blflcs75hf1qpzwfyi7zifc";
+      name = "qtwayland-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtwebchannel = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtwebchannel-everywhere-src-6.4.3.tar.xz";
-      sha256 = "1a7kpsy5c9vmwk69csnni6n6kn4zpvbf9fwbr1j4mrzhhx2h8mg9";
-      name = "qtwebchannel-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtwebchannel-everywhere-src-6.5.0.tar.xz";
+      sha256 = "1gzi03nqgaai0jjhy61gasdydkd0xpvrnq671671ns7kdmj3smfr";
+      name = "qtwebchannel-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtwebengine = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtwebengine-everywhere-src-6.4.3.tar.xz";
-      sha256 = "09995fhpzkpycjgad4s2wh5wx3vxl95h35cd3fj7kp516vvmmy2m";
-      name = "qtwebengine-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtwebengine-everywhere-src-6.5.0.tar.xz";
+      sha256 = "01vg60g25aabki4xlszfn3aq62yjbm2qdh0yy6gpwc0vlwsdl41a";
+      name = "qtwebengine-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtwebsockets = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtwebsockets-everywhere-src-6.4.3.tar.xz";
-      sha256 = "13s5im5ms7bza9f9dy6ahnxb5d9ndgvxfw83asp86pjwnmz3a9yy";
-      name = "qtwebsockets-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtwebsockets-everywhere-src-6.5.0.tar.xz";
+      sha256 = "0233c75by9n48hvm9wc8zmsry8ba0ckykdna1h9dld5vavb7n25w";
+      name = "qtwebsockets-everywhere-src-6.5.0.tar.xz";
     };
   };
   qtwebview = {
-    version = "6.4.3";
+    version = "6.5.0";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.4/6.4.3/submodules/qtwebview-everywhere-src-6.4.3.tar.xz";
-      sha256 = "0hz8ydf45nfxdsp2srff1yq2qpan50flwyw2aa4js52y95q1g5ai";
-      name = "qtwebview-everywhere-src-6.4.3.tar.xz";
+      url = "${mirror}/official_releases/qt/6.5/6.5.0/submodules/qtwebview-everywhere-src-6.5.0.tar.xz";
+      sha256 = "087hs60bzpgqs08z2prfbg9544mpii0xn4xkslp24zbaq43l5aq5";
+      name = "qtwebview-everywhere-src-6.5.0.tar.xz";
     };
   };
 }

